Collection&InterfacePractice/Exercise5/src/Main.java

```diff
@@ -0,0 +1,29 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Car car1 = new Car("Volkswagen Golf", 200000);
+        Car car2 = new Car("Toyota Corolla", 350000);
+        Car car3 = new Car("Ford Escort", 175000);
+        Car car4 = new Car("Toyota Corolla", 350000);
+        Car car5 = new Car("Toyota Corolla", 100000);
+
+        List<Car> cars = new ArrayList<>();
+
+        cars.add(car1);
+        cars.add(car2);
+        cars.add(car3);
+        cars.add(car4);
+        cars.add(car5);
+
+        cars = cars.stream().distinct().collect(Collectors.toList());
```

Comment:
Bien ahi que chusmeaste los streams, la semana que viene los vemos en profundidad, creo que la idea de mati aca era que intenten hacerlo a mano. Como recomendacion, si solo vas a imprimirlos por pantalla aprovecha el pipe y hacelo en el foreach, te ahorras la iteracion de abajo.
```suggestion
        cars.stream()
            .distinct()
            .forEach(System.out::println);
```

Reply:
Sisi ya habia chusmeado un poco por arriba los streams, pero si me parece que lo tendria que haber hecho de otra forma. Ahh buenisimo, mas simplificado y legible se hace asi. Gracias por la recomendacion Fausto.